sdks/java/io/cassandra/build.gradle

```diff
@@ -24,14 +24,14 @@ enableJavaPerformanceTesting()
 description = "Apache Beam :: SDKs :: Java :: IO :: Cassandra"
 ext.summary = "IO to read and write with Apache Cassandra database"
 
-def cassandra_version = "3.5.0"
+def cassandra_version = "3.6.0"
```

Comment:
I guess this definition could be deleted since it's not used anymore

Reply:
Good catch, fixing it, thanks.